modules/@apostrophecms/area/index.js

```diff
@@ -563,7 +564,8 @@ module.exports = {
           widgetEditors[name] = (browserData && browserData.components && browserData.components.widgetEditor) || 'AposWidgetEditor';
           widgetManagers[name] = manager.__meta.name;
           widgetIsContextual[name] = manager.options.contextual;
-          contextualWidgetDefaultData[name] = manager.options.defaultData;
+          widgetHasInitialModal[name] = manager.options.initialModal !== false; // TODO: use `skipInitialModal` to avoid doing `!== false`?
```

Comment:
use `skipInitialModal: true` rather than `initialModal` to reduce conditions complexity:

```js 
widgetSkipInitialModal[name] = manager.options.skipInitialModal === true;
```

default case is that nothing changes, by default we have the modal, except if we explicitly set `skipInitialModal` to `true`.
Seems more natural than saying "it has no initial modal"

WDYT?